aspnetcore/client-side/blazor/introduction/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Introduction to Blazor
+author: guardrex
+description: Learn how Blazor runs in the browser to execute C#/Razor code with WebAssembly and the Mono runtime in this introduction.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 03/26/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/introduction/index
+---
+# Introduction to Blazor
```

Comment:
This intro needs to be relevant to both existing .NET devs and web devs that are interested in trying out our platform.  there needs to be some text here about why you would want to use .NET in the browser and for full-stack development. See the "Why use .NET for browser apps?" section in https://blogs.msdn.microsoft.com/webdev/2018/02/06/blazor-experimental-project/ for the .NET value props we have been emphasizing.

Reply:
That's was why I linked out to the Blazor landing page from the docs landing page. I assumed this content would be over there and that the content would duplicated if I placed it here. That link is gone now, so I'll make it the first section here.

Reply:
The landing page will have swim lanes with some terse text, but I think it's worth elaborating a bit here.